### ISSUE TITLE
gmt which to download earth_relief_*.grd files

### ISSUE THREAD

**Issue by abanbj** [GenericMappingTools/gmt#432]
Trying to use ``gmt which`` to download ``earth_relief_*.grd``-files. The manual states (https://genericmappingtools.github.io/gmt/dev/gmtwhich.html) that this is done with the ``-G`` switch. However, I'm getting:

```
$ gmt which -G@earth_relief_20m.grd
gmtwhich [ERROR]: Download mode @earth_relief_20m.grd not recognized
gmtwhich [ERROR]: Syntax error: No files specified
```

**Comment by abanbj** [comment]
Was to soon. The right command is 

```
gmt which -Gu @earth_relief_10m.grd
```

**Comment by abanbj** [comment]
Another thing. The ``-G``-option seems to be ignored. The file is places in ``.gmt/server`` regardless of chosen ``-G[c|l|u]``

```
~ 10:33:08 $ gmt which -Gu @earth_relief_30m.grd
/home/user/.gmt/server/earth_relief_30m.grd
~ 10:33:20 $ gmt which -Gc @earth_relief_30m.grd
/home/user/.gmt/server/earth_relief_30m.grd
~ 10:33:22 $ gmt which -Gl @earth_relief_30m.grd
/home/user/.gmt/server/earth_relief_30m.grd
```

**Comment by PaulWessel** [comment]
I think it is mostly unclear documentation.  The -Gu will download and place it in the .gmt/server data directory.  The -Gl will place it in the current directory but NOT if the file already exists and is found in the server directory.  Delete the server file and run with -Gl and it will place a copy in your current directory.

**Comment by abanbj** [comment]
I see. It would be great to have a way to download and place the file in the current directory regardless of if it exists in the server directory.

**Comment by PaulWessel** [comment]
Implemented in PR #436, now closed.

**Comment by abanbj** [comment]
Now downloads to current folder with ``-Gl``. But, if the file exists in the current folder, issuing a ``gmt which -Gc @earth_relief_10m.grd`` does not download to ``~/.gmt/server``. 

```
$ gmt which -Gl @earth_relief_10m.grd
earth_relief_10m.grd
$ gmt which -Gc @earth_relief_10m.grd
earth_relief_10m.grd
```
If the file is not in the present folder ``-Gc`` workds as expected:

```
$ gmt which -Gc @earth_relief_10m.grd
/home/user/.gmt/server/earth_relief_10m.grd
```

Also, if using ``-Gu``, how is ``GMT_USER_DIR`` defined? I have problems finding documentation on this.

```
$ gmt --show-datadir
Not set
```
